src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,46 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMAL_AGE = 18;
+    private static final int MINIMAL_PASSWORD_LENGTH = 6;
+    private static final int FINAL_STATIC_ZERO = 0;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
+        if (checkAge(user) && checkLogin(user) && checkPassword(user)) {
+            storageDao.add(user);
+            return user;
+        }
         return null;
     }
+
+    private boolean checkAge(User user) {
```

Comment:
```suggestion
    private boolean checkAge(Integer age) {
```
краще передавати в метод безпосередньо значення а не викликати щоразу user.getAge()